DataFormats/HeavyIonEvent/interface/ClusterCompatibility.h

```diff
@@ -0,0 +1,34 @@
+#ifndef DataFormats_ClusterCompatibility_h
+#define DataFormats_ClusterCompatibility_h
+
+#include <vector>
+
+namespace reco { class ClusterCompatibility {
+public:
+
+  ClusterCompatibility();
+  virtual ~ClusterCompatibility();
+
+  int nValidPixelHits() const { return nValidPixelHits_; }
```

Comment:
please add some comments to the code to describe the meaning of the variables; use /// or //! for doxygen parsing